activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/main/resources/keycloak-client.properties

```diff
@@ -1,6 +1,6 @@
 activiti.cloud.services.oauth2.iam-name=keycloak
 
-spring.security.oauth2.client.registration.keycloak.client-id=${activiti.keycloak.client-id}
+spring.security.oauth2.client.registration.keycloak.client-id=${activity.keycloak.client-id}
```

Comment:
Why did you use `activity` with `y` at the end?

Reply:
it is a typo, thanks!